gnmi_server/constants_native.go

```diff
@@ -1,5 +1,6 @@
-// +build !gnmi_native_write
-
-package gnmi
-
-const ENABLE_NATIVE_WRITE = false
+//go:build !gnmi_native_write
```

Comment:
What is the reason to change? Is it by accident duplicate with next line? @ganglyu Could you review this line?

Reply:
[Build Tags - The Silver Gopher](https://andrewwphillips.github.io/blog/build-tags.html) Go fmt changes the format of the derives https://groups.google.com/g/golang-codereviews/c/GRxa5qejSPE, also see https://www.perplexity.ai/search/is-it-true-that-starting-with-2EwzsbTQSqiG6Kr709u7lw